DS3231.cpp

```diff
@@ -261,6 +261,11 @@ byte DS3231::getYear() {
 // setEpoch function gives the epoch as parameter and feeds the RTC
 // epoch = UnixTime and starts at 01.01.1970 00:00:00
 void DS3231::setEpoch(time_t epoch, bool flag_localtime) {
+	/*
+	 Arduino implementation of these `time` C++ functions
+	 is from Jan 1st, 2000, instead of Jan 1st, 1970.
+	*/
+	epoch -= SECONDS_FROM_1970_TO_2000; // Unix to y2k time
```

Comment:
Hi Andy,

do we distinguish between Arduino and ESP8266?
Or do you mean with with the Arduino time function implementation, the implementation of the DS3231?

Reply:
Which firmware `time` library works with ESP8266? Here is the link to the AVR library, which starts at 1/1/2000.

https://www.nongnu.org/avr-libc/user-manual/group__avr__time.html

If the same, great.
If not, we may need to do an architecture check.